pkg/utils/imagevector/types.go

```diff
@@ -14,6 +14,15 @@
 
 package imagevector
 
+const (
+	// Amd64 is a constant for the 'amd64' architecture.
+	Amd64 = "amd64"
+	// Arm64 is a constant for the 'arm64' architecture.
+	Arm64 = "arm64"
+	// WildcardArchitecture is a constant for multi architectures.
+	WildcardArchitecture = "*"
```

Comment:
can we reuse constants in `v1beta1constants` package?

Reply:
We should not, It will make imagevector package dependent on apis.

Reply:
And why would that be a bad thing?

Reply:
If someone wants to use image vector, apis package will be required for it which is big, just because of two constants.

Reply:
The image vector pkg is not a proper "exposed" API and it's not supposed to be used as such, that's why it's located under `pkg/utils`. Hence, we shouldn't care about someone importing this package.
Even if we would, that someone would probably import the API packages anyway. And even if not, the API packages are not very large and should be rather decoupled meanwhile.

TL;DR: please use the constants